app/src/org/commcare/appupdate/CommcareFlexibleAppUpdateManager.java

```diff
@@ -161,8 +161,6 @@ private void publishStatus() {
         if (mAppUpdateState == newState) {
             return;
         }
-        Logger.log(LogTypes.TYPE_CC_UPDATE, "Publishing status update to : " + newState.name() + ", from : "
-                + (mAppUpdateState != null ? mAppUpdateState.name() : "null"));
```

Comment:
@shubham1g5 Are we 100% sure that this is unnecessary? We can consider logging when the state changes.

Reply:
This already logs only when state changes (look at the if cond above this). I don't think this gives us any useful debugging info as such as most functionality there is managed by Play services API. Also there is nothing like `100% certainity` with what you might and might not need with logs and think we should avoid this framing in deciding what to log. Instead a better framing for retroactively deleting logs is have you found a particular line of useful while solving any debugging issues in fast ? Happy to re-consider if so.

Reply:
> Instead a better framing for retroactively deleting logs is have you found a particular line of useful while solving any debugging issues in past ?